index.html

```diff
@@ -0,0 +1,38 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Todo</title>
+</head>
+<body>
+<div class="todoapp">
```

Comment:
For wrappers better to use id. I'll explain you sometime.
and rename to 'todo-app'

Reply:
need for styles from node_modules